### ISSUE TITLE
commit message is not displayed when using svn

### ISSUE THREAD

**Issue by redguardtoo** [syohex/emacs-git-messenger#30]
I think follow code has not svn implementation:
```lisp
(defun git-messenger:git-commit-message (commit-id)
  (let ((args (git-messenger:cat-file-arguments commit-id)))
    (unless (zerop (git-messenger:execute-command 'git args t))
      (error "Failed: 'git cat-file'"))
    (goto-char (point-min))
    (forward-paragraph)
    (buffer-substring-no-properties (point) (point-max))))
```

I think you can use svn log -r changset
for example:
```bash
$ svn log -r 27990
------------------------------------------------------------------------
r27990 | C55520 | 2015-03-02 19:05:18 +1100 (周一, 02 三月 2015) | 1 line

Modified for US1880, US2301, US2323, US2887 
------------------------------------------------------------------------
```

**Comment by syohex** [comment]
`git-messenger:svn-commit-message` is called in subversion repository(`git-messenger:git-commit-message ` is not called in subversion repository)

Commit message is displayed as below.
![subversion](https://cloud.githubusercontent.com/assets/554281/6612270/89c872d4-c8bc-11e4-8dbd-672e25e68e26.png)

**Comment by redguardtoo** [comment]
looks it's because git-messenger:svn-message

I insert the log code in side it:
msg=------------------------------------------------------------------------
r28580 | C55520 | 2015-03-12 17:17:35 +1100 (Thu, 12 Mar 2015) | 1 line

 commit-id=28580
```

**Comment by syohex** [comment]
I have applied your fix at #33.
